spec/text_message_spec.rb

```diff
@@ -0,0 +1,6 @@
+require 'text_message'
+
+describe TextMessage do
```

Comment:
I wouldn't expect you to test an external framework, specifically, but maybe just to test your expectations of what you're getting from this framework. 

For example, if you're expecting a method called send_text in Order to call @text_message.send_message(total_cost), you would test this in order_spec.rb. 

eg. 
```
it 'sends a text to the user when they place their order' do
     expect(text_message).to receive(:send_message).with(1).argument
     order.send_text
end
```

It may seem quite simple, but it's testing the expected behaviour: this method is not responsible for the actual sending of the text, but it's responsible for _calling the method that does so._

There's one caveat to the above test being possible, and that caveat is that it requires us to be able to pass in a version of @text_message that we can set this 'expectation' on. Currently, your test would ignore our above 'expect', as it gets to line 11 of Order and it creates a new instance by running @text_message = TextMessage.now. 

We can get around this with dependency injection. This would mean refactoring your initialize method to look like this: 

```
def initialize(text_message = TextMessage.new)
     @ordered_items = []
     @menu = Menu.new
     @total = 0
     @text_message = text_message
end
```

This means that in your test, you would write it like this: 

```
it 'sends a text to the user when they place their order' do
     text_message = double
     allow(text_message).to receive(:send_message).with(1).argument

     order = Order.new(text_message)

     expect(text_message).to receive(:send_message).with(1).argument
     order.send_text
end
```